app/uk/gov/hmrc/brm/utils/BirthRegisterCountry.scala

```diff
@@ -26,6 +26,8 @@ object BirthRegisterCountry extends Enumeration {
   type BirthRegisterCountry = Value
   val ENGLAND = Value("england")
   val WALES = Value("wales")
+  val NORTHERN_IRELAND = Value("Northern Ireland")
```

Comment:
@chris-ianson @manishsw Do we feel it's the best approach to send through a text string here? Would it be safer to send through an "identifier" i.e. 1 = england 2 = wales or ENG WA, SCOT, NI?